docs/content/intro/quickstart.md

```diff
@@ -166,7 +165,7 @@ Install Brigade on your local development cluster. See our [Installation] instru
 Authenticate to Brigade as the root user using demo password `F00Bar!!!`. The -k flag instructs Brigade to ignore the self-signed certificate used by our local installation of Brigade.
 
 ```
-brig -k login --server https://localhost:8443 --root
+brig login -k --server https://localhost:8443 --root
```

Comment:
I use `-k` but I wonder if for clarity in docs we should use long-form flags? `-k` is short for `--insecure`.

Reply:
I like to use the long form of flags in documentation because it's more clear, and leave picking the short flags for people who are saving keystrokes. Let me fix that.